src/components/schedule-visualizer/tests/robots-overlay.test.tsx

```diff
@@ -6,21 +6,59 @@ import RobotsOverlay from '../robots-overlay';
 import ColorManager from '../colors';
 import Robot from '../robot';
 import fakeFleets from '../../../mock/data/fleets';
+import { createMuiTheme } from '@material-ui/core';
 
-test('Render robots correctly', () => {
-  const robots = fakeFleets()[0].robots;
-  const bounds = new L.LatLngBounds([0, 25.7], [-14, 0]);
-  const colorManager = new ColorManager();
-  // TextEncoder is not available in node
-  colorManager.robotColor = jest.fn(async () => 'black');
-  colorManager.robotColorFromCache = jest.fn(() => 'black');
-  const wrapper = mount(
-    <LMap>
-      <RobotsOverlay bounds={bounds} colorManager={colorManager} robots={robots} />
-    </LMap>,
-  );
-  expect(wrapper.find(Robot).exists()).toBeTruthy();
-  expect(wrapper.find(Robot).length).toBe(robots.length);
+const robots = fakeFleets()[0].robots;
+const bounds = new L.LatLngBounds([0, 25.7], [-14, 0]);
+const colorManager = new ColorManager();
+// TextEncoder is not available in node
+colorManager.robotColor = jest.fn(async () => 'black');
+colorManager.robotColorFromCache = jest.fn(() => 'black');
+let conflictRobotNames: string[][] = [];
 
-  wrapper.unmount();
+describe('Robots Overlay', () => {
+  const theme = createMuiTheme({
+    palette: {
+      error: {
+        main: '#f44336',
+      },
+    },
+  });
```

Comment:
I think we can use just `const theme = createMuiTheme();` since we are using the default theme.

Reply:
oh okay, i thought we would need to specify props for a new theme!